bin/describe/describe_workspace.ml

```diff
@@ -562,8 +562,13 @@ module Crawl = struct
                 (Context.build_dir context)
                 (Dune_file.dir dune_file)
             in
-            let project = Dune_file.project dune_file in
-            executables sctx ~options ~project ~dir exes
+            let* expander = Super_context.expander sctx ~dir in
```

Comment:
shouldn't this rather be in the `executables` function?

Reply:
Done in https://github.com/ocaml/dune/pull/10881/commits/90021898996766cebaeb7324f9c176dca9f56aaf.

Reply:
@anmonteiro could you have another look at this PR?

Reply:
ah yup, looks good. just approved